src/graphql/models/Cooccurrence.js

```diff
@@ -0,0 +1,43 @@
+import {
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLID,
+} from 'graphql';
+import Article from './Article';
+import Node from '../interfaces/Node';
+
+const Cooccurrence = new GraphQLObjectType({
+  name: 'Cooccurrence',
+  interfaces: [Node],
+  fields: () => ({
+    id: { type: new GraphQLNonNull(GraphQLID) },
+    userId: { type: GraphQLNonNull(GraphQLString) },
+    appId: { type: GraphQLNonNull(GraphQLString) },
+    articles: {
+      type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(Article))),
+      resolve: async ({ articleIds }, args, { loaders }) =>
+        loaders.searchResultLoader.load({
+          index: 'articles',
+          type: 'doc',
+          body: {
+            query: {
+              ids: {
+                values: articleIds,
+              },
+            },
+          },
+        }),
+    },
+    articleIds: {
+      type: new GraphQLNonNull(
+        new GraphQLList(new GraphQLNonNull(GraphQLString))
+      ),
+    },
+    createdAt: { type: GraphQLString },
+    updatedAt: { type: GraphQLString },
```

Comment:
From the API that creates this model, it seems that `createdAt` and `updatedAt` always exists. We can then add `GraphQLNonNull` on these fields with confident.

```suggestion
    createdAt: { type: new GraphQLNonNull(GraphQLString) },
    updatedAt: { type: new GraphQLNonNull(GraphQLString) },
```

Reply:
updated